core/curl_script.sh

```diff
@@ -1,31 +1,12 @@
 #!/bin/bash
-let count=0
-
+let name
+name="signin.html"
 while IFS= read -r line; do
-    # echo $count
-    if (( $count % 2 == 0 ))
+    if [[ $line == *"https://"* ]]
     then
-        v2=$(echo -e "$line" | tr -d '[:space:]')
-        v2="$v2.mp4"
+        curl $line -o $name
```

Comment:
The curl command needs headers here to keep the connection alive while all the bits are transferred
Otherwise, it may stop in the middle, if the file size is huge.